imagetool/src/main/java/com/oracle/weblogic/imagetool/cli/ImageTool.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
new file?  should be 2020 only

Reply:
Is it considered a new file if I rename an old one?

Reply:
I guess not